app/services/reports/base_report.rb

```diff
@@ -4,6 +4,15 @@ module Reports
   class BaseReport
     private
 
+    def fiscal_start_date
+      now = Time.zone.now
+      if now.strftime('%m').to_i >= 10
```

Comment:
What if we just access the `.month` property and save some string parsing, and use `.change` to simplify the string parts?

```ruby
today = Time.zone.today
if today.month >= 10
  today.change(month: 10, day: 1).to_s
else
  today.change(month: 10, day: 1, year: today.year - 1).to_s
end
```

Reply:
This doesn't put the date in postgres format: month-day-year

Reply:
I could put a strftime on that but it looks a lot more complicated to me

Reply:
i like the now.month

Reply:
what do you think of the latest?

Reply:
1. `psql` seems to accept both?

  ```psql
  =# SELECT '2020-1-15'::date AS year_month_day, '1-15-2020'::date AS month_day_year;
   year_month_day | month_day_year 
  ----------------+----------------
   2020-01-15     | 2020-01-15
  (1 row)
  ```

2. Have we considered trying to use the ORM to convert/escape dates for us via placeholders or explicitly?
  ```ruby
   t = Time.zone.now
   ActiveRecord::Base.connection.quote(t)
  ```

Reply:
Ah you are right about the date format!

Reply:
Are you still liking the prior vs this?
```
  if now.month >= 10
      now.strftime('10-01-%Y')
  else
      "10-01-#{now.year - 1}"
```

Reply:
Personally, yes I think it's much better to handle date objects as objects rather than strings, and think format strings are hard to read, but am OK if you feel strongly about writing out the components like this